docs/platform-integration/device/geocoding.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Geocoding"
+description: "Learn how to use the .NET MAUI Geocoding class in the Microsoft.Maui.Devices.Sensors namespace. This class provides APIs to both geocode a placemark to a positional coordinate, and reverse geocode coordinates to a placemark."
+ms.date: 05/12/2022
+no-loc: ["Microsoft.Maui", "Microsoft.Maui.Devices", "Microsoft.Maui.Devices.Sensors"]
+---
+
+# Geocoding
+
+This article describes how you can use the .NET Multi-platform App UI (.NET MAUI) `IGeocoding` interface. This interfaces provides APIs to geocode a placemark to a positional coordinates and reverse geocode coordinates to a placemark. The `IGeocoding` interface is exposed through the `Geocoding.Default` property.
+
+[!INCLUDE [docs under construction](~/includes/preview-note.md)]
+
+The `Geocoding` and `IGeocoding` types are available in the `Microsoft.Maui.Devices.Sensors` namespace.
+
+## Get started
+
+To access the **Geocoding** functionality the following platform-specific setup is required.
+
+<!-- markdownlint-disable MD025 -->
+# [Android](#tab/android)
+
+No setup is required.
+
+# [iOS](#tab/ios)
+
+No setup is required.
+
+# [Windows](#tab/windows)
+
+A Bing Maps API key is required to use geocoding functionality. Sign up for a free [Bing Maps](https://www.bingmapsportal.com/) account. Under **My account** > **My keys**, create a new key and fill out information based on your application type, which should be **Windows Application**.
+
+Early on in your application's life before calling any `Geocoding` methods, set the API key, which is only available on Windows:
```

Comment:
This sentence and code example needs updating. While it's true, the new approach is to use `ConfigureEssentials` and specify your map token in `UseMapServiceToken`. Ultimately it just sets `Platform.MapServiceToken` but it's now the recommended approach.